src/StatusQ/Components/StatusChatToolBar.qml

```diff
@@ -135,43 +135,10 @@ Item {
                          (menuButton.visible || membersButton.visible || searchButton.visible)
             }
 
-            StatusFlatRoundButton {
+            StatusActivityCenterButton {
```

Comment:
Just out of curiosity: Is there more places where `StatusActivityCenterButton` is used?

Reply:
minimum 2 :)